src/shrutiDJun20ExceptionHandling/ItemIDValue.java

```diff
@@ -0,0 +1,45 @@
+package shrutiDJun20ExceptionHandling;
+
+public class ItemIDValue {
+
+	void fetchItemID(String url, String iD) { // taking String name as iD so
+												// that in future if EMP id is
+												// needed, code can be reused
+												// with minor changes
+
+		String urlafterId = "";
+		String itemId = "";
+		if (url.contains("itemId=")) {
+			urlafterId = url.substring((url.indexOf(iD) + iD.length()));
+
+			for (int index = 0; index < urlafterId.length(); index++)
```

Comment:
Good job dear, 
Suggesting an alternative You can also write : 
try{
String str = input.split("itemId=")[1].replace("/", "").split("&")[0];
System.out.println("Item id is " + str);

}catch(StringIndexOutOfBoundsException siob){
			System.out.println("Item ID -> Not present");
		}

This two line in try block and it will work.